rocm-llvm-mlir/PKGBUILD

```diff
@@ -0,0 +1,39 @@
+# Maintainer: JP-Ellis <josh@jpellis.me>
+
```

Comment:
Do you mind adding me as co-maintainer as I will be pushing it to the AUR?

Reply:
Sure 😁